File: thematic_tutorials/bhax_textbook/codes/2. hét/parentchild.java
public class Szulo{
public class Gyerek extends Szulo{
 public void kiir(){System.out.println("Gyerek");}
 }
public void main(String[] args)
 {
 Szulo szulo = new Gyerek();
 System.out.println(szulo.kiir());
 }

File: thematic_tutorials/bhax_textbook/codes/2. hét/parentchild.cpp
#include <iostream>
using namespace std;
class Szulo
{};
class Gyerek: public Szulo{
public: void kiir(){
cout<<"gyerek"<<endl;
}
};
int main()
{
Szulo* sz= new Gyerek();
cout << sz->kiir() <<endl;
}